app/views/projects/_project.html.erb

```diff
@@ -31,6 +31,11 @@
           </li>
         <% end %>
       <% end %>
+      <% if project.stages.count > 8 %>
```

Comment:
also this is doing 3 queries now ... select + count + count
cheapest would be to do `.first(max+1).size == max` to detect if to display a `More` link or not (without indicator)